lib/src/base-styles/_nx-page-layout.scss

```diff
@@ -83,7 +90,7 @@
   }
 
   .nx-page {
-    min-width: $nx-page-width-min - $nx-scrollbar-width;
+    min-width: var(--page-width-max) - $nx-scrollbar-width;
```

Comment:
I'd also expect this to need to be a calc

Reply:
> I'd also expect this to need to be a calc

Same as above the units are the same. `1600px - 15px`

Reply:
FYI I fixed the min/max issue above here: https://github.com/sonatype/sonatype-react-shared-components/pull/328/commits/484bdde0327df578e0d550f1200762378eb06c7b

Reply:
> Same as above the units are the same. 1600px - 15px

That doesn't matter; CSS only does math in `calc` expressions.  If you try to do math without one that would be SCSS math, which of course can't work with runtime values like a `var()`.  See this screenshot for proof that the `min-width` is not working:

![image](https://user-images.githubusercontent.com/3630091/111156083-b5060e80-856b-11eb-9b23-aaa64aee5b42.png)

Reply:
> > Same as above the units are the same. 1600px - 15px
> 
> That doesn't matter; CSS only does math in `calc` expressions. If you try to do math without one that would be SCSS math, which of course can't work with runtime values like a `var()`. See this screenshot for proof that the `min-width` is not working:
> 

I came to the same conclusion this weekend but for different reasons, namely what would happen if a consumer set one of the variables to a different unit. So I made the change here: https://github.com/sonatype/sonatype-react-shared-components/pull/328/commits/92071b851ee12d9c3139fe32eb6c39b80aa3c008

Note that I had to move the scrollbar width into a CSS var or the calc for page scrolling wouldn't work in real time.

Reply:
> Note that I had to move the scrollbar width into a CSS var or the calc for page scrolling wouldn't work in real time.

That doesn't sound right to me, but I'm not against move that to a CSS variable.

Reply:
> > Note that I had to move the scrollbar width into a CSS var or the calc for page scrolling wouldn't work in real time.
> 
> That doesn't sound right to me, but I'm not against move that to a CSS variable.

It's weird, I couldn't see any reason that it shouldn't work but in the inspector the `min-width` (the result of the  `calc()`) was always `0`